middleware/statsd/statsd_middleware.py

```diff
@@ -0,0 +1,48 @@
+from datadog import DogStatsd
+from django.utils.deprecation import MiddlewareMixin
```

Comment:
@burnerlee please take a look at other files to see how imports are organized. `import` statements go first and then we list the `from import`. This is not correct